src/commands/drop.ts

```diff
@@ -19,17 +20,23 @@ const command: CommandDefinition = {
 					channel: msg.channel.id,
 					message: msg.id,
 					user: msg.author.id,
+					server: msg.guildID,
 					tournament: id,
+					tournamentServer: participant?.tournament.owningDiscordServer,
 					command: "drop",
 					...payload
 				})
 			);
 		}
 		log({ event: "attempt" });
-		if (!participant) {
+		if (!participant || (msg.guildID && msg.guildID !== participant.tournament.owningDiscordServer)) {
 			await reply(msg, `You are not signed up for __${id}__ or that tournament doesn't exist!`);
 			return;
 		}
+		if (participant.tournament.status === TournamentStatus.COMPLETE) {
```

Comment:
Not mutating a tournament after it's done is a change we've been meaning to make, but is it in the scope of this PR? Will changes to other methods of dropping be made in this PR?

Reply:
It's not in scope, this was just convenient to do.